setup.py

```diff
@@ -24,5 +24,10 @@
         'watchdog',
         'semantic_version',
         'prompt_toolkit>=2.0'
+    ],
+    classifiers=[
+        'Programming Language :: Python :: 3.5',
+        'Programming Language :: Python :: 3.6'
```

Comment:
No it doesn't, it's just for the benefit of users on PyPI. From https://pypi.org/classifiers/:
> Each project's maintainers provide PyPI with a list of "trove classifiers" to categorize each release, describing who it's for, what systems it can run on, and how mature it is.
>
> These standardized classifiers can then be used by community members to find projects based on their desired criteria.

Reply:
in which case I'll merge this.

Reply:
I haven't tested `sml-sync` with 3.7.0rc1, which is currently the latest RC, but can do so and add 3.7 here.